app-crypt/certbot/certbot-0.40.1.ebuild

```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-PYTHON_COMPAT=(python{3_6,3_7})
+PYTHON_COMPAT=(python{3_6,3_7,3_8})
```

Comment:
please just edit the latest version and the live ebuilds (so 1.2.0-r1 and 9999 in this case).